website/content/api-docs/system/plugins-reload.mdx

```diff
@@ -0,0 +1,121 @@
+---
+layout: api
+page_title: /sys/plugins/reload - HTTP API
+description: The `/sys/plugins/reload` endpoints are used to reload plugins.
+---
+
+# `/sys/plugins/reload`
+
+## Reload plugin
+
+The `/sys/plugins/reload/:type/:name` endpoint reloads a named plugin across all
+namespaces. It is only available in the root namespace. All instances of the plugin
+will be killed, and any newly pinned version of the plugin will be started in
+their place.
+
+| Method | Path                              |
+| :----- | :-------------------------------- |
+| `POST` | `/sys/plugins/reload/:type/:name` |
+
+### Parameters
+
+- `type` `(string: <required>)` – The type of the plugin, as registered in the
+  plugin catalog. One of "auth", "secret", or "database".
```

Comment:
Do we want to list the "unknown" type here as well?

Reply:
Good point 👍